Tests/Kernels/NetworkTest/Kernel.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Cosmos.HAL;
+using Cosmos.System.Network;
+using Cosmos.System.Network.ARP;
+using Cosmos.System.Network.Config;
+using Cosmos.System.Network.IPv4;
+using Cosmos.System.Network.IPv4.UDP;
+using Cosmos.System.Network.IPv4.UDP.DHCP;
+using Cosmos.System.Network.IPv4.UDP.DNS;
+using Cosmos.TestRunner;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Sys = Cosmos.System;
+
+namespace NetworkTest
+{
+    public class Kernel : Sys.Kernel
+    {
+        protected override void BeforeRun()
+        {
+        }
+
+        protected override void Run()
+        {
+            /** Ethernet Packet Parsing Test **/
+            byte[] ethernetPacketData = new byte[]
```

Comment:
Is it maybe possible to create tests which just instantiate clients (if the driver exists) and then closes them again?